test/verify/test_isinf.cpp

```diff
@@ -49,4 +49,5 @@ struct test_isinf : verify_program<test_isinf<T>>
 };
 
 template struct test_isinf<migraphx::half>;
+// template struct test_isinf<migraphx::bf16>;
```

Comment:
There is a place to disable the tests just for the cpu backend here:

https://github.com/ROCm/AMDMIGraphX/blob/develop/test/verify/main.cpp#L70

So we can still get coverage for the GPU.